### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by RichardBongani**

### CODE REVIEW

**Comment by shshamim63** [comment]
## [!!Status!!]:red_circle: Changes Required :red_circle: 
___
Hi @RichardBongani and @pyven :wave: ,
Great initiative ;clap. You have gathered all the item to decorate the page. Just need to play with them a little bit more to declare this project as complete.
___
##### Please follow the instructions below :point_down:  to complete this project.
___
### :spades: [Mandatory Changes] :spades: 
___

1. The stickler is not working like it should be. Can you check this [link](https://github.com/microverseinc/linters-config/tree/master/css) again to ensure that you have done it correctly. Especially the point no. 1 & 2.
#### :large_blue_diamond:  [Large Version]
1. The latest funding section should contain space to the right side. Well the actual page contains it. So, can you give a try to fix it.
task1
2. App, gear, Tech section also do not containing space on the right side. And the text are rendering above the image. Well, I think it is because of the width declaration. Some are above 100%.
task2
3. latest deals section also having the same issue. The problems solution relies on the width.
task3
#### :large_blue_circle: [Medium version]
1. Latest news section is not possessing the place it should. The third item of each row is only showing almost like 50%. It is because of the width declaration.
task4
2. Same as above, not contain the items properly.
task5
3. Does not fit on the medium screen properly. And the other parts too.
task6
4. Latest deal also having the same issue.
task7
#### :large_blue_diamond:  [Mobile Version]
1. The top 3 sections text are not fitting properly.
task8
2. Latest news is also having the same issue.
task9
3. Rest of the sections on the mobile view are also possessing the width like it should. Check the actual website to figure out what could be the cause.
___
### Please Fix the above issues and send another code review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

**Comment by trox115** [comment]
**Hello @RichardBongani  and @pyven** , very nice to meet you :handshake: . 
I think you should be proud of your work. However, this project still needs some improvements that you probably forgot to make when requested.

**1- Your project still has some problems for medium size, as you can see :**

medium2
medium3

_Please correct this._

**2- The large size still need some work too.**

large1 

**3- It is a bad practice to hard code this kind of things, please remove the `<br>` and use your styles instead.** 
hardcoded.

**4- Avoid using the !important on your CSS. Here is a [link](https://speckyboy.com/good-bad-css-practices/) with CSS best practices**


**Your Mobile looks really good congratulations!** 

Come on Guys, one more little effort and you are ready to go :muscle:  . Do not forget to validate the project through [W3C Validator](https://validator.w3.org/#validate_by_input). 

## Sumary ## 

- [ ] I have corrected all container problems with medium sizes.
- [ ] I have corrected all container problems with large sizes.
- [ ] I have removed all `<br>` and I used the styles.

# Status: Changes Required # 

P.S - If you have any problem you can reach me through Slack. My name is António Fernandes

**Comment by pelzolga123** [comment]
## Hi Team  
#### Your project looks :+1: good! But it steel needs some corrections.

* [ ] Large size containers look not like an original. Please make changes like it is shown below.
1. Images in 'main-cover' div need to be separated. 'Latest news' section needs to be in the center of the page. Also you need to add some space between rows in this section.
image

Original: 
image

2. 'Latest funding rounds' section also needs to be in the center, 'Get updates by email', 'Add Info' and 'See More' links need to be aligned like in original.
image

Original: 
image

3. Next section also needs to be in center, but also it would be good to align ul with pictures and add some spacing between h2 and image in this section. Take a look at original.
image

Original: 
image

4. 'Latest deals' section also needs to be formatted like an original one.
image

Original: 
image

5. Please, add some padding to footer to make it look better.

### Kindly make changes and submit another code review request.

If you need help or have some questions please find me on slack. *Happy coding*

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project needs to look more like an original one, layout needs corrections.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1297 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    overflow-x: hidden;
```
Comment (suretrust) [inline style.css]: This hidden property is affecting your design on medium and small screens.

hidden

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1297 @@
...
+    right: 50px;
+}
+
+.sm-nav {
+    display: none;
+}
+
+@media only screen and (max-width: 992px) {
+    html {
+        overflow-x: hidden;
```
Comment (suretrust) [inline style.css]: This is not a good way to style this. It will disrupt your design

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1297 @@
...
+    .own_games,
+    .tech-img,
+    .my-gear,
+    .my_netflix{
+        width: 305px;
+        margin: 0 30px 0 30px;
+    }
+
+    html {
+        overflow-x: hidden;
```
Comment (suretrust) [inline style.css]: This is not a good way to style this. It will disrupt your design

File: index.html [hunk 4]
```diff
@@ -0,0 +1,604 @@
...
+            <a href="#"><i class="fas fa-bars" id="menu"></i></a>
+            <a href="#"><i class="fas fa-search" id="sm-search"></i></a>
+        </nav>
+    <header class="nav-container">
+        <nav>
+        <div class="tnw-gif">
+        <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="TNW">
+        </div>
+        <section class="nav-sub-container">
+        <div class="navigation1">
```
Comment (suretrust) [inline index.html:25]: Your design is having double vertical scroll bars, let's get rid of the inner one. 

vert

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @RichardBongani and @pyven,

### Great job thus far, :raised_hands: however, there are still some loopholes in your design.

- [ ] Please see my comments on your code.

- [ ] Your design is not transitioning properly in between screen sizes.

- [ ] I suggest you reset your page before you start styling it. You could use [this](https://meyerweb.com/eric/tools/css/reset/) or just the code below:

`body {
    padding: 0;
    margin: 0;
    box-sizing: border-box;
}`

**Kindly submit for another code review once these issues have been fixed.** _Happy coding!_ :tada:

**Comment by shshamim63** [comment]
## Status :tada: Approved :tada: 
___
Hi @RichardBongani  and @pyven :wave: ,
Nice work :clap. Most of the part are at the place just it should be. But there are still some room that you can do to improve the project. Like, APP, GEAR sections margin and padding. Before merging you have to do a mandatory task. 
### :red_circle: Mandatory :red_circle: 
___
##### Please enable your stickler. It is not working properly. Follow this [link](https://github.com/microverseinc/linters-config/tree/master/css) to enable it and fix the errors that you will get after enabling it. If you find difficulty to enable it, feel free to contact with me. 
___
#### Please follow the instruction and after that you can merge it. Best of luck for your next project
___
## :clubs:  Happy Coding :club:
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

**Pull request merged**